spack-repo/packages/singularity-eos/package.py

```diff
@@ -77,13 +96,18 @@ class SingularityEos(CMakePackage, CudaPackage):
         when="+cuda",
     )
 
+    for _myver,_kver in zip(("@:1.6.2","@1.7.0:"),("@3.2:","@3.3:")):
+        depends_on("kokkos" + _kver, when=_myver)
+        depends_on("kokkos-kernels" + _kver, when=_myver)
+
     # set up kokkos offloading dependencies
     for _flag in ("~cuda", "+cuda", "~openmp", "+openmp"):
-        depends_on("kokkos@3.2: ~shared" + _flag, when="+kokkos" + _flag)
-        depends_on("kokkos-kernels@3.2:" + _flag, when="+kokkos-kernels" + _flag)
+        depends_on("kokkos ~shared" + _flag, when="+kokkos" + _flag)
+        depends_on("kokkos-kernels" + _flag, when="+kokkos-kernels" + _flag)
         depends_on("spiner" + _flag, when="+kokkos" + _flag)
 
     # specfic specs when using GPU/cuda offloading
+    # TODO Do we need `+aggressive_vectorization`, `+cuda_constexpr`, `~compiler_warnings` ?
     depends_on("kokkos +wrapper+cuda_lambda+cuda_relocatable_device_code", when="+cuda+kokkos")
```

Comment:
how did I let this slip by. I thought we remove rdc from all of our spackages @mauneyc-LANL.

Reply:
I don't recall doing anything with this; it's been in there for some time 

https://github.com/lanl/singularity-eos/blob/b5d7d8cd5c8525cc9d51a71102a645b9c1df6d6e/spack-repo/packages/singularity-eos/package.py#L80C9-L80C9